sdk/cognitivelanguage/azure-ai-language-questionanswering/migration_guide.md

```diff
@@ -60,70 +69,381 @@ client = QuestionAnsweringClient(
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.language.questionanswering.authoring import QuestionAnsweringAuthoringClient
+
+client = QuestionAnsweringAuthoringClient(
+    endpoint="https://<my-cognitiveservices-account>.cognitiveservices.azure.com",
+    credential=AzureKeyCredential("API key")
+)
+```
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.ai.language.questionanswering.authoring import QuestionAnsweringAuthoringClient
+
+client = QuestionAnsweringAuthoringClient(
+    endpoint="https://<my-cognitiveservices-account>.cognitiveservices.azure.com",
+    credential=DefaultAzureCredential()
+)
+```
+
+#### Creating knowledge base
+
+Previously in `azure-cognitiveservices-knowledge-qnamaker`, you could create a new knowledgebase using a `CreateKbDTO`:
+
+```python
+from msrest.authentication import CognitiveServicesCredentials
+from azure.cognitiveservices.knowledge.qnamaker.models import CreateKbDTO, QnADTO
+from azure.cognitiveservices.knowledge.qnamaker import QnAMakerClient
+
+client = QnAMakerClient(
+    endpoint="https://<my-cognitiveservices-account>.cognitiveservices.azure.com",
+    credentials=CognitiveServicesCredentials("API key")
+)
+
+operation = client.knowledgebase.create(
+    create_kb_payload=CreateKbDTO(
+        name="<knowledgebase-name>",
+        qna_list=[
+            QnADTO(
+                questions=["questions"],
+                answer="answer"
+            )
+        ]
+        
+    )
+)
+```
+
+Now in `azure-ai-language-questionanswering`, you can create a new Question Answering project by passing a `dict` with the needed project creation properties:
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.language.questionanswering.authoring import QuestionAnsweringAuthoringClient
+
+client = QuestionAnsweringAuthoringClient(
+    endpoint="https://<my-cognitiveservices-account>.cognitiveservices.azure.com",
+    credential=AzureKeyCredential("API key")
+)
+
+project = client.create_project(
+    project_name="<project_name>",
+    options={
+        "description": "This is the description for a test project",
+        "language": "en",
+        "multilingualResource": True,
+        "settings": {
+            "defaultAnswer": "no answer"
+        }
+    })
+```
+
+#### Updating knowledge base
+
+Previously in `azure-cognitiveservices-knowledge-qnamaker`, you could update your knowledge base using a `UpdateKbOperationDTO`:
+
+```python
+from msrest.authentication import CognitiveServicesCredentials
+from azure.cognitiveservices.knowledge.qnamaker.models import UpdateKbOperationDTO, UpdateKbOperationDTOAdd, QnADTO
+from azure.cognitiveservices.knowledge.qnamaker import QnAMakerClient
+
+client = QnAMakerClient(
+    endpoint="https://<my-cognitiveservices-account>.cognitiveservices.azure.com",
+    credentials=CognitiveServicesCredentials("API key")
+)
+
+operation = client.knowledgebase.update(
```

Comment:
Same here. Just seems incomplete without getting the same results. How would I make the jump from the old to the new?

Reply:
updated